### PR TITLE
Iar 23/tomas/create a posts feed be

### PR DESCRIPTION
**Pull request by TomasBenes**

### CODE REVIEW

File: src/controllers/PostsController.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+	@Get('/')
+	getPostsFeed(): Promise<Posts[]> {
+		return this.postsService.findAll();
+	}
+
+	/*
+	@Post('/subreddits/posts/create')
+	createPost(@BodyParams() posts: Posts): Promise<void> {
+		return this.postsService.create(posts);
+	}*/
```
Comment (Karhalll) [inline src/controllers/PostsController.ts]: Preparation for future i guess but keep in mind that in apirary this endpoint shoud be /api/v1/subreddits/posts/create but now is under /api/v1/feed/subreddits/posts/create

File: src/controllers/UserController.ts [hunk 2]
```diff
@@ -16,4 +16,5 @@ export class UserController {
 	logInUser(@BodyParams() user: UserLogin): Promise<void> {
 		return this.userService.logIn(user);
 	}
+
```
Comment (Karhalll) [inline src/controllers/UserController.ts]: Is it realy nessesary to add emtpy lines on end of classes?

File: src/entities/Posts.ts [hunk 3]
```diff
@@ -15,6 +15,7 @@ export class Posts {
 	date_created: Date;
 
 	@ManyToOne(type => Subreddits, subreddit => subreddit.posts)
+	@Column()
```
Comment (Karhalll) [inline src/entities/Posts.ts:18]: Why? You already create column in database with ManyToOne notation. Why we need another column in database table?

**Review by Karhalll — APPROVED** [review]
Overall good code but again. From my point of view the unused code will raise some issues (at least more questions) for future implementation.

**Comment by SimProch** [comment]
There are couple outstanding questions from Martin that are valid and have no reaction. I'm moving this back to team code review.

**Review by SimProch — APPROVED** [review]

**Pull request merged**